packages/inngest/src/types.ts

```diff
@@ -304,7 +304,19 @@ export type AnyHandler = Handler<any, any, any, any>;
  */
 export interface EventPayload {
   /**
-   * A unique identifier for the event
+   * A unique id used to idempotently ingest a given event payload once (i.e.
+   * deduplication).
+   *
+   * Set this when sending events to ensure that the event is only processed
+   * once; if an event with the same ID is sent again, it will be ignored.
```

Comment:
```suggestion
   * A unique id used to idempotently process a given event payload.
   *
   * Set this when sending events to ensure that the event is only processed
   * once; if an event with the same ID is sent again, it will not invoke functions.
```

Reply:
The aim here is to make it clear that funciton runs are handled idempotently.  Handling idempotency at the event level is too much work for the events API, which handles tens of thousands of events per second 😬

Reply:
(ty - committed this)